client-side/src/stories/popup/popup.scss

```diff
@@ -0,0 +1,45 @@
+@import '../variables.scss';
+
+.ReactModal__Overlay {
+    background-color: rgba(0, 0, 0, 0.5);
+}
+  
+.ReactModal__Content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: $standardInfoColor;
+    border-radius: 1%; 
+    padding: 2%; 
+    width: 80%;
+    max-width: 60%; 
+    box-shadow: 0 4px 8px $primary-color;
+}
+  
+.modal-header {
+    font-size: 2%; 
+    margin-bottom: 1%; 
+}
+  
+.modal-body {
+    margin-bottom: 2%;
+}
+  
+.modal-footer {
+    display: flex;
+    justify-content: flex-end;
+}
+  
+.modal-button {
+    background-color: $primary-color;
+    border: none;
+    color: $standardInfoColor;
+    padding: 1%; 
+    border-radius: 0.5%;
+    cursor: pointer;
+}
+  
+.modal-button.close {
+    background-color: $secondary-color;
```

Comment:
please update this file to scss hierarchy, instead of css structure

Reply:
you should have one parent class, and all the styles should be inside